apps/juicebox_web/assets/js/app.scss

```diff
@@ -1,5 +1,4 @@
-.app {
-  p {
-    color: red;
-  }
+body {
+  margin: 0;
```

Comment:
Properties should be ordered padding, margin

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)